addons/pos_restaurant/models/pos_session.py

```diff
@@ -67,21 +67,24 @@ def get_onboarding_data(self):
     def _load_onboarding_data(self):
         super()._load_onboarding_data()
         convert.convert_file(self.env, 'pos_restaurant', 'data/pos_restaurant_onboarding.xml', None, mode='init', kind='data')
-        restaurant_config = self.env.ref('pos_restaurant.pos_config_main_restaurant')
-        if len(restaurant_config.session_ids.filtered(lambda s: s.state == 'opened')) == 0:
-            self.env['pos.session'].create({
-                'config_id': restaurant_config.id,
-                'user_id': self.env.ref('base.user_admin').id,
-            })
-        convert.convert_file(self.env, 'pos_restaurant', 'data/pos_restaurant_onboarding_open_session.xml', None, mode='init', kind='data')
+        restaurant_config = self.env.ref('pos_restaurant.pos_config_main_restaurant', raise_if_not_found=False)
+        if restaurant_config:
+            convert.convert_file(self.env, 'pos_restaurant', 'data/pos_restaurant_onboarding_main_config.xml', None, mode='init', kind='data')
+            if len(restaurant_config.session_ids.filtered(lambda s: s.state == 'opened')) == 0:
+                self.env['pos.session'].create({
+                    'config_id': restaurant_config.id,
+                    'user_id': self.env.ref('base.user_admin').id,
+                })
+            convert.convert_file(self.env, 'pos_restaurant', 'data/pos_restaurant_onboarding_open_session.xml', None, mode='init', kind='data')
 
     def _after_load_onboarding_data(self):
         super()._after_load_onboarding_data()
-        configs = self.config_id.filtered('module_pos_restaurant').union(self.env.ref('pos_restaurant.pos_config_main_restaurant', raise_if_not_found=False))
-        configs.with_context(bypass_categories_forbidden_change=True).write({
-            'limit_categories': True,
-            'iface_available_categ_ids': [Command.link(self.env.ref('pos_restaurant.food').id), Command.link(self.env.ref('pos_restaurant.drinks').id)]
-        })
+        configs = self.config_id.filtered('module_pos_restaurant')
+        if configs:
+            configs.with_context(bypass_categories_forbidden_change=True).write({
+                'limit_categories': True,
+                'iface_available_categ_ids': [Command.link(self.env.ref('pos_restaurant.food').id), Command.link(self.env.ref('pos_restaurant.drinks').id)]
+            })
```

Comment:
the same comment above https://github.com/odoo/odoo/pull/138035/files#r1350367936